tests/e2e/kcs/test_mon_crash_recovery_scenario.py

```diff
@@ -0,0 +1,126 @@
+import logging
+import pytest
+from random import choice
+from threading import Thread
+
+from ocs_ci.ocs.constants import (
+    ROOK_CEPH_OPERATOR,
+    CEPHBLOCKPOOL,
+    STATUS_CLBO,
+)
+from ocs_ci.helpers.helpers import modify_deployment_replica_count
+from ocs_ci.ocs.resources.deployment import get_mon_deployments
+from ocs_ci.ocs.resources.pvc import get_pvc_objs
+from ocs_ci.ocs.resources.pod import get_ceph_tools_pod
+from ocs_ci.ocs.resources.storage_cluster import ceph_mon_dump
+from ocs_ci.framework.pytest_customization.marks import tier3
+from ocs_ci.ocs.defaults import OCS_OPERATOR_NAME
+from ocs_ci.helpers.helpers import wait_for_resource_state
+
+
+log = logging.getLogger(__name__)
+
+
```

Comment:
this test is not valid for external mode deployment, we may have to skip it

Reply:
Added @skipif_external_mode marker to the test.